dependency/catalog_nodes_test.go

```diff
@@ -239,6 +263,18 @@ func TestCatalogNodesQuery_Fetch(t *testing.T) {
 							//"consul-network-segment": "",
 						},
 					},
+					{
+						Node:            "node" + tenancy.Partition,
+						Address:         testConsul.Config.Bind,
+						Datacenter:      "dc1",
+						TaggedAddresses: map[string]string{
+							//"lan": "127.0.0.1",
```

Comment:
Since this is commented.  I feel that we should remove it.

Reply:
we could, but I want to keep it to investigate why it's not working intermittently.